libs/search/src/lib/facets/facets-container/facets-container.component.ts

```diff
@@ -1,4 +1,9 @@
 import { Component, Input, OnInit } from '@angular/core'
+import { select, Store } from '@ngrx/store'
+import { map } from 'rxjs/operators'
+import { SearchState } from '../../state/reducer'
+import { getSearchResultsAggregations } from '../../state/selectors'
+import { FacetsService } from '../facets.service'
```

Comment:
Would it make sense to use Aggregations or Facets everywhere for consistency ?

If yes, I would vote for Aggregation. 

Components in https://github.com/geonetwork/geonetwork-ui/pull/38 could be Aggs > Agg > AggBuckets > AggBucket

Reply:
`aggregations` is a way elastic aggregates the data, `facet` is a UI representation, other components like `metrics` can use aggregations: I would keep the distanciation.